docs/fundamentals/networking/quic/quic-overview.md

```diff
@@ -0,0 +1,317 @@
+---
+title: QUIC support in .NET
+description: Learn about the support for QUIC protocol in .NET.
+ms.date: 03/15/2023
+helpviewer_keywords:
+  - "protocols, QUIC"
+  - "sending data, QUIC"
+  - "QUIC"
+  - "receiving data, QUIC"
+  - "application protocols, QUIC"
+  - "Internet, QUIC"
+---
+# QUIC
+
+QUIC is a new, transport layer protocol. It has been recently standardized in [RFC 9000](https://www.rfc-editor.org/rfc/rfc9000). It uses UDP as an underlying protocol and it's inherently secure as it mandates TLS 1.3 usage, see [RFC 9001](https://www.rfc-editor.org/rfc/rfc9001). Another interesting difference from well-known transport protocols such as TCP and UDP is that it has stream multiplexing built-in on the transport layer. This allows having multiple, concurrent, independent data streams that do not affect each other.
```

Comment:
Just a thought, maybe it will be better to reword as "QUIC is a transport layer protocol standardized in [RFC 9000](https://www.rfc-editor.org/rfc/rfc9000)." -- given it's a doc page that will potentially stay for a long time when "recently" might no longer apply 😄
(I see the same "new" wording in H/3 doc as well, so if you'd agree with my proposal, I think it might be better to remove "new" from there as well)
WDYT?